go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/f-ewald/adr
```

Comment:
@f-ewald Would it be possible to put the /marouni/ path here ?

Reply:
Yes, I will change this.

Reply:
Thx @f-ewald 
I'll rebuild test and get back to you ASAP